app/push-sw.js

```diff
@@ -16,9 +16,8 @@ self.addEventListener('push', (evt) => {
   console.log('Push received ', obj);
 
   if (obj && obj.message) {
```

Comment:
we should use obj.action instead of obj.message in the future, ping @JohanLorenzo